charlesinto/iReporter
# #162578040 Delete a red-flag record

#### What does this PR do?

- Delete a red-flag record

#### Description of Task to be completed?
- Implement jwt authentication
- Verify the record was posted by the user
- Delete the record from the database

#### How should this be manually tested?
- Clone the repo - `git clone  git repo address`
- Enter into the project directory - `cd repo folder`
- switch to this branch - `PR branch`
-  Start  the frontend.
- Navigate to `endpoint`
- Open `Postman`
- Grab your token from the application tab in Developers tab
- On postman add the token in headers
- - On `postman` Send a `DELETE` request to `http://localhost:7000/api/v1/red-flags/:id`
- You should receive a successful response

#### Any background context you want to provide?
- the POST endpoint response should be structured like this- 


#### What are the relevant pivotal tracker stories?
[#162532134 ](https://www.pivotaltracker.com/story/show/162532134 )

#### Screenshots (if appropriate)

![image](https://user-images.githubusercontent.com/26615085/49808052-ddcf4600-fd5b-11e8-9fe2-9297b5d8f5ab.png)




#### Questions:
N/A
